tests/integration/local/invoke/test_integrations_cli.py

```diff
@@ -439,7 +439,7 @@ def test_invoke_returns_expected_results_from_git_function_with_parameters(self)
 
 
 class TestSamInstrinsicsAndPlugins(InvokeIntegBase):
-    template = "template-pseudo-params.yaml"
+    template = Path("template-pseudo-params.yaml")
```

Comment:
All other subclasses of InvokeIntegBase assign `template` a `Path` value, except for this. Good catch by mypy